### DISCUSSION TITLE
"Cannot be booked at same time" GPLv3 v2.8.6.1

### DISCUSSION THREAD

**Discussion by daj1u06** [LibreBooking/librebooking#514] [Q&A]
The help wiki has the following section:

### Preventing Resources From Being Booked at the Same Time
Resource Relationships can be used to define resources that cannot be booked at the same time. To configure this, add a resource within the Cannot Be Booked at the Same Time section. Once set, all non-administrative users will be prevented from booking a resource if any other related resource is booked at the same time.

Where is this option toggled, I can't find it anywhere?

Thanks.

**Comment by daj1u06** [comment]
![resource](https://github.com/user-attachments/assets/32a072a0-a9d3-4660-ba3d-41bb8042d0e6)

This is what I see for any resource, there is no "cannot be booked at same time option".

**Comment by hachuah** [comment]
I grepped through the codebase, and I can't find it as well. Perhaps the wiki lines were taken from the commercial bookedscheduler software, which has this feature? Did you eventually solve your problem? As I'm looking for a solution to this as well.
